README.md

```diff
@@ -1,15 +1,25 @@
 # BSK [![Build Status](https://travis-ci.org/socialpoint-labs/bsk.svg?branch=master)](https://travis-ci.org/socialpoint-labs/bsk)
 
+## Packages
+
+### Extensions
```

Comment:
I would generalize them as "packages", explaining when they add new important features or just when they are extensions to the standard libraries features. What do you think?